src/sdk/namespace_fs.js

```diff
@@ -1495,6 +1500,7 @@ class NamespaceFS {
                 config.NSFS_OPEN_READ_MODE,
                 get_umasked_mode(config.BASE_MODE_FILE),
             );
+            // @ts-ignore the next call is incorrect...
```

Comment:
I think you can just remove this function completely

Reply:
done